File: 20140504/fizzbuzz.c
#include <stdio.h>

static void
fizzbuzz (min, max)
{
  int i;
  for (i = min; i <= max; i++)
    {
      if (i % 15 == 0)
        printf ("FizzBuzz\n");
      else if (i % 3 == 0)
        printf ("Fizz\n");
      else if (i % 5 == 0)
        printf ("Buzz\n");
      else
        printf ("%d\n", i);
    }
}

int
main (void)
{
  fizzbuzz (1, 100);
}

File: 20140617/fizzbuzz.py
def toStr(i):
    if i % 15 == 0:
        return "FizzBuzz"
    elif i % 3 == 0:
        return "Fizz"
    elif i % 5 == 0:
        return "Buzz"
    else:
        return i

def fizzBuzz(min,max):
    for i in range(min,max+1):
        print(toStr(i))
    
fizzBuzz(1,100)
